gen/plugins/com.salesforce/com.salesforce.plugin.smartstore.js

```diff
@@ -158,61 +158,82 @@ var buildSmartQuerySpec = function (smartSql, pageSize) {
     return inst;
 };
 
+// Helper function to handle calls missing isGlobalStore argument
+// Return true if isGlobalStore argument was missing and calling function was re-invoked with the additional isGlobalStore argument
+// Return false if isglobalstore argument is present
+var callWithGlobalStoreArgument = function(argumentsOfCaller) {
```

Comment:
thanks to that function, the first argument (isGlobalStore) can be optional: when no boolean is passed in, we re-invoke the target function with false as first argument